src/attr.rs

```diff
@@ -192,6 +196,10 @@ impl Attribute {
     /// *This function is available if Syn is built with the `"parsing"`
     /// feature.*
     #[cfg(feature = "parsing")]
+    #[cfg_attr(
+        syn_enable_doc_cfg,
+        doc(cfg(all(any(feature = "derive", feature = "full"), feature = "parsing")))
```

Comment:
I added that on purpose, so that a user, who searches for that function name, immediately sees all conditions of the function itself.

Should it be simplified (reduced to `parsing` only)?

Reply:
I'd prefer to only show `parsing`, as the method will be displayed on the page for the type, which already has the `"derive"` and `"full"` feature annotations on it. 

We generally don't repeat these cfg annotations on other methods on cfg-limited types, so it seems inconsistent and noisy to do it here.